src/main/java/nextstep/subway/facade/PathFacade.java

```diff
@@ -0,0 +1,50 @@
+package nextstep.subway.facade;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import nextstep.subway.entity.Section;
+import nextstep.subway.entity.Station;
+import nextstep.subway.entity.group.SectionGroup;
+import nextstep.subway.service.SectionService;
+import nextstep.subway.service.StationService;
+import nextstep.subway.service.response.PathResponse;
+import nextstep.subway.service.response.StationResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class PathFacade {
+
+    private final SectionService sectionService;
+    private final StationService stationService;
+
+    public PathFacade(SectionService sectionService, StationService stationService) {
+        this.sectionService = sectionService;
+        this.stationService = stationService;
+    }
+
+    public PathResponse getPath(long source, long target) {
+
+        validateDontEquals(source, target);
+
+        final Station start = stationService.findById(source);
+        final Station finish = stationService.findById(target);
+
+        final List<Section> sectionList = sectionService.findAll();
+        final SectionGroup sectionGroup = SectionGroup.of(sectionList);
+
+        return new PathResponse(
+            sectionGroup.getPath(start, finish).stream()
+                .map(StationResponse::of)
+                .collect(Collectors.toList()),
+            sectionGroup.getPathDistance(start, finish)
+        );
```

Comment:
지난 피드백이 반영이 안된 것 같아요 😢 
SectionGroup은 Line이 가지는 구간들을 관리하는 객체이니 경로계산을 위한 Path 객체를 만들어보시면 어떨까요?
```suggestion
        final List<Section> sectionList = sectionService.findAll();
        final Path path = new Path(sectionList, start, finish);

        return new PathResponse(path.getStations(), path.getDistance());
```

```java
public class Path {
    private final WeightedMultigraph<Station, DefaultWeightedEdge> graph;
    private final GraphPath<Station, DefaultWeightedEdge> graphPath;

    public Path(List<Section> sections, Station start, Station finish) {
        ...
        graphPath = new DijkstraShortestPath<>(graph).getPath(start, finish);
    }

    public List<Station> getStations() {
    ...
    }
    
    public List<Station> getDistance() {
    ...
    }
}
```

Reply:
section List 에 계속 집중했었는데 생각해보니 Path 객체는 일급컬렉션일 필요가 전혀없었네요
상세한 예시 감사드립니다 !!!!!